database/migrations/20240915183745-create-vehicles-table.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Vehicles', {
```

Comment:
This table should include a country column in regard to the Norwegian and other potential future integrations

Reply:
fixed